### DISCUSSION TITLE
Image segmentation with a U-Net-like architecture

### DISCUSSION THREAD

**Discussion by kkmcgg** [NSCC-COGS/Aestheta#35] [Ideas]
[Image segmentation with a U-Net-like architecture](https://keras.io/examples/vision/oxford_pets_image_segmentation/)

The natural progression to move beyond our simple pixel based gradient boost classifier is to attempt to train a deep neural network for image segmentation. This can be done relatively successfully and somewhat simply using the keras library. The above tutorial shows a pretty good strep by step approach to achieving this using training samples of 7400 images. So, we need some satellite training data!
